### ISSUE TITLE
Problem in CSS parsing

### ISSUE THREAD

**Issue by meichengg** [etigerstudio/typora-misty-theme#7]
Hi, first of all, i want to say i really enjoyed your theme, been using it for months, really awesome. Thanks

I usually convert my Markdown to PDF file, and when i have some texts in bold, after converting to PDF using Typora of course, those bold words aren't bold, i think because of small font-weight, can you fix this please?
I also tried to change font-weight of \<strong\> tag in the CSS file, but it became super thick 😭 

I'm using Ubuntu 18.04, latest Typora and Misty Theme Macos

**Comment by etigerstudio** [comment]
🥰Glad that you enjoyed this little theme!

In your description above, you said you got unexpectedly thin font weight when exporting Markdown document to PDF from Typora.
Font faces that come with Ubuntu distros might be hugely different from either Windows or macOS, while Misty Theme is mainly tuned for modern Windows or macOS releases. So, the theme CSS as current version regardless of Win or Mac might not be 100% usable & beautiful on Ubuntu.
However, I would still suggest you to try Misty Theme **Windows version** as bold font weight in Mac version CSS is set to **500** whereas that in Win is set to **600** which is a little bolder.
Sometimes, especially for simpler fonts which only includes limited font weight variants such as some Linux built-in **Chinese** font, font weight of 500(Medium) could be preferably rendered as 400(Normal) using the [Fallback weights](https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight#Fallback_weights) system while font weight of 600(Semi Bold) could be preferably rendered as 700(Bold).
This might be the reason that caused this problem.

**TL;DR: you could just try switching to Windows version of the theme to see if anything gets better.**

If this cannot solve the problem effectively, feel free to contact me at that time and you could post some screenshots & other information to help us further inspect the problem if necessary.

**Comment by etigerstudio** [comment]
Probably fixed in aa0bcdd14f34ec846256146af6d99408fc4723ca.
Latest v1.0-alpha included that fix.
